docs/monthly/2024-01-monthly.md

```diff
@@ -12,150 +12,97 @@ developments to gather their feedback on proposed plans.
 
-This representation is equivalent to the TextEnvelope produced by cardano-cli,
-except with an additional `txId` field, that is guaranteed to match the transaction
-id within the cbor object.
+This representation mirrors the TextEnvelope produced by the Cardano CLI, with the addition of an `txId` field, which is guaranteed to match the transaction ID within the CBOR object.
 
-:::warning Breaking Change
+:::warning Breaking change
 
-We do not plan to go through a deprecation cycle on this breaking change to the API. If you
-are affected and see a problem updating your Hydra client to use CBOR, please
-[open an issue](https://github.com/input-output-hk/hydra/issues/new) on Github.
+We do not plan to go through a deprecation cycle on this breaking change to the API. If you are affected and see a problem updating your Hydra client to use CBOR, please [open an issue](https://github.com/input-output-hk/hydra/issues/new) on GitHub.
 
 :::
 
 ### Hydra Chess
-We started building a game on Hydra as dogfooding allows us to find things which
-are suboptimal or stumble accross a bug that needs fixing. Hydra Chess proved to
-be no different and we learned in the process of making this dApp.
+
+The team began developing a game on Hydra, leveraging dogfooding to uncover suboptimal elements or stumble upon bugs requiring resolution. Hydra Chess exemplified this process, offering valuable lessons during DApp development.
 
 ![](https://ipfs.io/ipfs/bafybeicxcm4yuedetm45kn6xrzqsc4mn2aocmhqtt6wrwxz5lzfry722ra/hydra-chess.png)
 
-As we iron out things we hope to improve the workflow of running `hydra-node` as
-part of a full peer-to-peer dApp. The goal is that it is easy enough to be run
-by non-tech savvy users, but also provide an example for people trying to build
-on Hydra. You can find the source code
-[here](https://github.com/abailly-iohk/hydra-chess).
+While refining the details, the team aims to improve the workflow of running `hydra-node` within a full peer-to-peer DApp framework. The objective is to ensure user-friendly accessibility while also providing an example for Hydra developers. You can access the source code [here](https://github.com/abailly-iohk/hydra-chess).
 
 ### Hydra Explorer
-To measure the progress and success of Hydra, we require tools that provide
-insights into its usage. For this purpose, we have initiated work on
-[#696](https://github.com/input-output-hk/hydra/issues/696) to enable anyone
-tracking Hydra heads across the whole life-cycle and observe the growth of the
-Hydra ecosystem.
 
-In this initial phase, we have implemented a basic backend service
+Measuring the progress and success of Hydra requires tools that provide
+insights into its usage. For this, the team initiated work on
```

Comment:
@olgahryniuk @oduameh The first sentence was better before IMO (isn't this one lacking a verb?)

Reply:
I rewrote it to drop the use of 'we'. The first sentence was good otherwise.